build/validated-languages.json

```diff
@@ -0,0 +1,30 @@
+[
+    "bs",
```

Comment:
This is a list of languages that have been manually validated - by me only, so probably not 100% accurate.